manifests/init.pp

```diff
@@ -362,6 +368,7 @@
   validate_bool($service_hasstatus)
   validate_bool($service_hasrestart)
   validate_bool($openmanage_enable)
+  validate_bool($snmpv2_enable)
```

Comment:
please do not add the legacy validate_* methods. Datatypes are the new way to go \o/